src/shutdown/shutdown.c

```diff
@@ -305,6 +343,15 @@ int main(int argc, char *argv[]) {
         (void) cg_get_root_path(&cgroup);
         in_container = detect_container() > 0;
 
+        /* If the logging target is not the console, and if we are not running from a container, then try
+         * to update the sysctl kernel.printk current value to a log level equal to 6 in order to see
+         * "notice" messages; This is only done if the current log level was at a lower value.
+         */
+        if (!IN_SET(log_get_target(), LOG_TARGET_CONSOLE, LOG_TARGET_CONSOLE_PREFIXED) &&
```

Comment:
hmm, wouldn't it make more sense to check for the log targets that actually use KMSG?